src/main/java/uk/gov/digital/ho/hocs/casework/client/notifyclient/NotifyClient.java

```diff
@@ -71,6 +77,17 @@ private void sendUserEmailCommand(UserAssignChangeCommand command){
         }
     }
 
+    @Retryable(maxAttemptsExpression = "${retry.maxAttempts}", backoff = @Backoff(delayExpression = "${retry.delay}"))
+    private void sendOfflineQaUserCommand(OfflineQaUserCommand command){
+        try {
+            Map<String, Object> queueHeaders = getQueueHeaders();
+            producerTemplate.sendBodyAndHeaders(notifyQueue, objectMapper.writeValueAsString(command), queueHeaders);
+            log.info("Sent Offline QA Email of type {} for Case UUID: {}, correlationID: {}, UserID: {}", command.getCommand(), command.getCaseUUID(), requestData.correlationId(), requestData.userId(), value(EVENT, OFFLINE_QA_EMAIL_SENT));
+        } catch (Exception e) {
+            log.error("Failed to send Email for case UUID {}", command.getCaseUUID(), value(EVENT, AUDIT_FAILED), value(EXCEPTION, e));
```

Comment:
We should use 'NOTIFY_EMAIL_FAILED' key here, methods above have the same issue.